core/modules/layout/css/layout.admin.css

```diff
@@ -27,18 +29,25 @@
   line-height: 1.3;
   max-width: calc(100vw - 60px); /* Fix flex-wrap issue in some browsers */
 }
+.layout-settings-page .layout-options.no-flexbox .form-checkboxes,
 .layout-settings-form .layout-options.no-flexbox .form-radios {
   display: block;
 }
+.layout-settings-page .layout-options .form-type-checkbox,
 .layout-settings-form .layout-options .form-type-radio {
   width: 100px;
   margin: 0 5px 5px 0;
   padding: 0;
   text-align: center;
 }
+.layout-settings-page .layout-options.no-flexbox .form-type-checkbox,
 .layout-settings-form .layout-options.no-flexbox .form-type-radio {
   float: left;
 }
+.layout-settings-page .layout-options .form-item input.form-checkbox {
+  /* needed to override style.css */
```

Comment:
Just elaborate that this is overriding *Seven* style.css, as opposed to some other theme.